tests/integration/julia/sandbox-mj.html

```diff
@@ -519,7 +531,7 @@
             <span data-f-bind="salesSq[4]"></span>.
         </td>
         <td class="pass">
-            pass (0.9.0). note that this only works for span, not for input (i'm assuming that's by design).
+            pass (0.10.0). note that this only works for span, not for input (i'm assuming that's by design).
```

Comment:
I get errors when I try to use converters with inputs, e.g. something like `<input data-f-bind="salesSq | first"></input>` to update the last element of the salesSq array. 

I'm assuming that's on purpose, but if not let me know and I can log an enhancement to make converters work with inputs.